examples/angular2/readme.md

```diff
@@ -0,0 +1,10 @@
+# Angular2 TodoMVC Example
+
+> Angular is a development platform for building mobile and desktop web applications
+
+Angular is a development platform for creating applications using modern web standards. Angular includes a wealth of essential features such as mobile gestures, animations, filtering, routing, data binding, security, internationalization, and beautiful UI components. It's extremely modular, lightweight, and easy to learn.
+
+### Run & Dev
+
+* `npm i`
+* `npm run dev`
```

Comment:
Let's build the readme like every other app: https://github.com/tastejs/todomvc-app-template/blob/master/app-readme.md :) Mainly I'd like to see some resources regarding Angular 2 and how to follow up on its development.